### PR TITLE
Re-serialising all test sets

### LINKED ISSUE #288: Re-serialise test sets
#### Description:

All testsets are to be updated by running them through the versioning toolkit.

All testset are still de-serialising fine, with the use of versioning, but should get them up to date so that versioning is not needed to be called every time and so that any future changes are more easily handled

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### Issues addressed by this PR

 Closes #288 

 

Updating all test-sets by turning them through the versioning toolkit and serialising them out again.

They were all de-serialising before, but should keep these up to date.

 ### Test files

https://burohappold.sharepoint.com/:f:/s/BHoM/Ekjy6DnHUihOtkoMtrwv-pYBbCGgpeL5CP9FOxYGe37zDQ?e=mr8dKc

 ### Changelog

 ### Additional comments

### CODE REVIEW

**Review by FraserGreenroyd — APPROVED** [review]
I have no qualms with the changes here, but good to get a review from @peterjamesnugent or @JosefTaylor before merging :smile:

**Review by peterjamesnugent — APPROVED** [review]
Serialises for me, I've had a look at the properties that are objects and they are serialising too. 

The method `TestDataOfType` does return a warning for every single dataset though:
`1. No Source information is available for the dataset named TestSets\Structure\Bar\01 - Straight planar bars`

That can be tackled in a separate PR.

**Comment by IsakNaslundBh** [comment]
> The method TestDataOfType does return a warning for every single dataset though:
>1. No Source information is available for the dataset named TestSets\Structure\Bar\01 - Straight planar bars

yeah, was aware, and agree, needs fixing, but out of scope. Thanks!

**Pull request merged**
